src/dndbeyond/content-scripts/character.js

```diff
@@ -857,24 +857,97 @@ function displayAction(paneClass) {
     });
 }
 
+function handleCustomText(paneClass) {
+    // Function Objective: Search for and handle block of VTT Message text...
+    //...in pane Notes or Description (Priority descending in that order.)
+    const regexp = /```\[([^\]]*)\]\s*((\S|\s)*?)\s*```/g;
+    const rollOrderTypes = ["before", "after", "replace"];//(relative to normal roll msg)
+    const defaultRollFunc = function () {};
+    let rollOrder = rollOrderTypes[0];
+    let rollFunc = defaultRollFunc;
+    
+    let pane = $("."+paneClass);
+    let elemText;
+    let matchedText;
+    
+    // Search for msg text in "Notes" section, if it exists
+    //Notes: Iterate through classes in pane: "ddbc-property-list__property" that contain the text "Notes:"
+    //  If exists: load text from "ddbc-property-list__property-content" (Last child of "ddbc-property-list__property")
+    let msgElement = pane.find(".ddbc-property-list__property:contains('Notes:')").get(0)
+    if (msgElement!=undefined && msgElement!=null) {
+        elemText = msgElement.lastChild.innerText;
+        // console.log("Notes Text: "+elemText);
+        matchedText = [...elemText.matchAll(regexp)];
+    }
+
+    // If no message was found, search for msg in the "Description" section, if it exists
+    if (matchedText==undefined || matchedText==null) {
+        msgElement = pane.find(".ct-action-detail__description").get(0);
+        console.log(msgElement);
+        if (msgElement!=undefined) {
+            elemText = msgElement.innerText;
+            // console.log("Description Text: "+elemText);    
+            matchedText = [...elemText.matchAll(regexp)]; 
+        }
+    }
```

Comment:
Seems you're only looking for `.ct-action-detail__description`, there's also `.ct-spell-detail__description` and same for item, also using `ddbc` instead of `ct`. Look for `__description` in the file to see the other things that could match. Along with previous notes search, I'd rewrite it like this : 
```js
const pane = $(`.${paneClass}`);
const notes = descriptionToString(pane.find(".ddbc-property-list__property:contains('Notes:')")
const notesMatch = notes.match(regexp);
const description = descriptionToString(pane.find(".ct-action-detail__description, .ct-spell-detail__description, .ct-item-detail__description, .ddbc-action-detail__description, .ddbc-spell-detail__description, .ddbc-item-detail__description,")
const descriptionMatch = description.match(regexp);
const customMacro = notesMatch || descriptionMatch; // Use this
```